### PR TITLE
docs(testing): improve cypress headless mode guide

### PR DESCRIPTION
**Pull request by santoshyadavdev**
## Related Issue(s)

Fixes #

### CODE REVIEW

**Comment by meeroslav** [comment]
Hey @santoshyadavdev, thank you for the PR. Can you please change the commit message prefix to `docs(testing):` and force push again to fix the build?

File: docs/shared/cypress-plugin.md [hunk 1]
```diff
@@ -63,9 +63,7 @@ Cypress doesn't currently re-run your tests after changes are made to applicatio
 
 ### Using Cypress in the Headless Mode
```
Comment (meeroslav) [inline docs/shared/cypress-plugin.md:64]: Can you please change the title to `Using Cypress in the Headed Mode` and modify the text slightly?

```
Cypress runs in headless mode by default (the `headless` option is deprecated). In order to run the Cypress in Headed mode, you can set `browser` option to select the specific browser to be used or set `headed` option to run tests using the default browser.
```

**Review by meeroslav — CHANGES_REQUESTED** [review]
Please check my comment for minor text improvement

File: docs/shared/cypress-plugin.md (hunk 1 above)
Comment (santoshyadavdev) [reply docs/shared/cypress-plugin.md]: Done.

**Review by meeroslav — APPROVED** [review]

**Pull request merged**